services/spark-operator/1.2.3/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - spark-operator.yaml
+  - ../../../helm-repositories/mesosphere-spark-repo
```

Comment:
I noticed this HelmRepository is also referenced [here](https://github.com/mesosphere/dkp-catalog-applications/blob/main/helm-repositories/mesosphere-spark-repo/kustomization.yaml), just out of curiosity does it need to be referenced here as well?

Reply:
We are referencing the `HelmRepository` by `../../../helm-repositories/mesosphere-spark-repo`. I am new to kustomize but my current understanding is that the link you posted makes it so that we can just throw the whole directory here. kind of like __init__.py for a python module.